content/cpp/concepts/math-functions/terms/sinh/sinh.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'sinh()'
+Description: 'Returns the hyperbolic sine of an argument given in radians.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`sinh()`** function returns the hyperbolic sine of an argument given in radians.
+
+## Syntax
+
+```cpp
+sinh(angle)
+```
+
+The `angle` parameter should be expressed in radians. To convert from degrees to radians:
```

Comment:
We seem to be missing the other side of the colon `:`?

Reply:
Done, Kindly check